packages/types/src/interfaces.ts

```diff
@@ -34,7 +34,8 @@ export interface Store {
 }
 
 export interface CosmosBlock {
-  block: Block;
+  blockId: BlockId;
+  block: {id: string} & Block;
```

Comment:
We need to ensure backwards compatibility. So the Tendermint block should be a superset of stargate block

Reply:
tendermint block is a superset of stargate block when `id` field is added to it.

```
//stargate:
export interface Block {
  /** The ID is a hash of the block header (uppercase hex) */
  readonly id: string;
  readonly header: BlockHeader;
  /** Array of raw transactions */
  readonly txs: readonly Uint8Array[];
}

//tendermint:
export interface Block {
  readonly header: Header;
  /**
   * For the block at height 1, last commit is not set.
   */
  readonly lastCommit: Commit | null;
  readonly txs: readonly Uint8Array[];
  readonly evidence: readonly Evidence[];
}
```

header is backward compatible